### ISSUE TITLE
Expose un-healthy nodes in info

### ISSUE THREAD

**Issue by aluzzardi** [docker/swarm#1136]
`docker info` should expose un-healthy nodes along with an error message.

Pseudo-Example:
```
Nodes: 1
 dev: 10.0.0.123:4242
  └ Status: Down
  └ Error: 10.0.0.123: No route to host
```

Thoughts?

**Comment by abronan** [comment]
:+1: Good idea as it helps troubleshoot connectivity issues to Swarm nodes. Most of the time we would think it's a Swarm bug (in discovery for example) while in reality it might just be a misconfigured firewall or an unexposed port.

**Comment by chanwit** [comment]
It's a great step towards cluster management APIs that you have mentioned earlier.

**Comment by UserTaken** [comment]
When using a token with dozens of registered machines, what if you just want to show the active nodes instead of every entry? When using a backend such as Etcd, inactive nodes are automatically removed after a set TTL so how would this work with that?

**Comment by jimmyxian** [comment]
:+1: More easy to find what's wrong.

**Comment by aluzzardi** [comment]
Merged into the node management proposal (#1486)
